src/abstractdataframe/show.jl

```diff
@@ -278,13 +278,28 @@ function _show(io::IO,
     # case, we will add this information using the row name column of
     # PrettyTables.jl. Otherwise, we can just use the row number column.
     if (rowid === nothing) || (ncol(df) == 0)
-        show_row_number = true
+        show_row_number::Bool = haskey(kwargs, :show_row_number) ? kwargs[:show_row_number] : true
```

Comment:
I assume kwargs supports `get`?:
```suggestion
        show_row_number::Bool = get(kwargs, :show_row_number, true)
```